EpiserverRedirects/Menu/HostStore.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using EPiServer.Shell.Services.Rest;
+using EPiServer.Web;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Forte.EpiserverRedirects.Menu;
+
+[RestStore("HostStore")]
+public class HostStore : RestControllerBase
+{
+    private readonly ISiteDefinitionRepository _siteDefinitionRepository;
+    private IEnumerable<HostDto> _allHosts;
+    private  List<HostDto> _options;
+
+    public HostStore(ISiteDefinitionRepository siteDefinitionRepository)
+    { 
+        _siteDefinitionRepository = siteDefinitionRepository;
+    }
+
+    public List<HostDto> Options
+    {
+        get {return _options ??= new List<HostDto>()
+        {
+            new HostDto() { Id = "1", Name = "All" }, //no filter
```

Comment:
Couple things here:
- I think it's better to make HostDto's property Id of type Guid so we don't need to use some "0" and "1" magic strings here.
- I think we can merge declaration and instantiation of this field in one line 
- in this approach we don't need aditional _options field
- maybe we can make HostDto a record instead of a class?


So I guess after his changes it might look something like this:
```
private readonly HostDto _allDto = new(Guid.Parse("5DAE2426-814E-496A-9099-1B53517A85C9"), "All");
    private readonly HostDto _allHostDto = new(Guid.Parse("7F757A26-1B69-486A-B335-41472ABE724A"), "All hosts");

    private readonly IList<HostDto> _options;

    public HostStore(ISiteDefinitionRepository siteDefinitionRepository)
    {
        _siteDefinitionRepository = siteDefinitionRepository;
        _options = new List<HostDto> {_allDto, _allHostDto};
    }
```